app/views/users/show.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
\ No newline at end of file
+<div class ='show'>
+    <div class = 'usersContent'>
+        <div class = 'each_user'>
+            <img src = <%= @user.photo %> class = 'image'>
+            <div class = 'name'>
+                <p><%= @user.name %></p>
+                <a>Number od Posts : <%= @user.posts_counter %> </a>
+            </div>
+        </div>
+    </div>
+    <div class='bio'>
+        <p>Bio : <br/><%= @user.bio %></p>
+    </div>
+</div>
+<div class='posts'>
+       <div class= 'post_section'>
+             <% @posts.each do |post| %>
+            <div class = 'each_post'>
+                <div class = 'post'>
+                    <p>Post#<%= post.id %> </p>
+                    <p><%= post.title %></p>
+                    <p><%= post.text %></p>
+                    <a>Number of Comments : <%= post.comments_counter %> </a>
+                </div>
+            </div>
+         <% end %>
```

Comment:
- [ ] As required for the project, kindly make sure you use the `#three_most_recent_posts` method to render the user's 3 most recent post in order to follow the [format given](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) in the project requirements. And when clicking on each `Post`, it takes you to its show page as shown in the **[wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)**.
```ruby
# Example

<% @user.three_most_recent_posts.each do |post| %>
  ...
<% end %>
```